src/general/LimitOrderHook.sol

```diff
@@ -309,9 +254,9 @@ contract LimitOrderHook is BaseHook, IUnlockCallback {
             orderInfo.liquidity[msg.sender] += liquidity;
         }
 
-        // unlock the callback to the poolManager, the callback will trigger `unlockCallback`
-        // note that multiple functions trigger `unlockCallback`, so the callbackData.callbackType will determine what happens
-        // in `unlockCallback`. In this case, it will add liquiidty out of range.
+        // unlock the callback to the poolManager, the callback will trigger `{unlockCallback}`
+        // note that multiple functions trigger `{unlockCallback}`, so the `callbackData.callbackType` will determine what happens
+        // in `{unlockCallback}`. In this case, it will add liquidity out of range.
```

Comment:
Internal comments are not picked up by the documentation rendering, so using `{}` in those is not required